tools/mo/requirements.txt

```diff
@@ -4,7 +4,7 @@ mxnet~=1.2.0; sys_platform == 'win32'
 mxnet>=1.7.0.post2,<=1.9.1; sys_platform != 'win32'
 networkx~=2.5; python_version <= "3.6"
 networkx<=2.8.8; python_version > "3.6"
-protobuf>=3.18.1,<4.0.0
+protobuf>=3.18.1,<5.0.0
```

Comment:
why do we have protobuf here? I see `google.protobuf` module usage only in files related to Caffe

Reply:
Good question, I will check and return.

Reply:
https://stackoverflow.com/questions/31308812/no-module-named-google-protobuf
Looks that it is the same. If you use `google.protobuf`, you need to install `protobuf`. But we can think of how to eliminate this dependency because TF has sort of similar API to work with protobuf or TF format files so no need to use low level API for parsing such files.